.github/workflows/ci-cd.yml

```
@@ -0,0 +1,324 @@
+
+    - name: Run tests
+      env:
+        DB_HOST: localhost
+        DB_PORT: 3306
+        DB_USER: root
+        DB_PASSWORD: testpassword
+        DB_NAME: hotaku_test_db
+        JWT_SECRET: test-secret-key
+      run: go test -v -race ./...
+
+    # TODO: Coverage reporting temporarily disabled
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)